lib/main_common.pm

```diff
@@ -1685,11 +1686,17 @@ sub load_toolchain_tests {
     loadtest "console/kdump_and_crash" if is_sle && kdump_is_applicable;
 }
 
+sub load_nvmftests {
+    return unless consolestep_is_applicable();
+    loadtest "kernel/nvmftests";
+}
+
 sub load_common_opensuse_sle_tests {
     load_autoyast_clone_tests           if get_var("CLONE_SYSTEM");
     load_create_hdd_tests               if get_var("STORE_HDD_1") || get_var("PUBLISH_HDD_1");
     load_toolchain_tests                if get_var("TCM") || check_var("ADDONS", "tcm");
     loadtest 'console/network_hostname' if get_var('NETWORK_CONFIGURATION');
+    load_nvmftests                      if get_var('NVMFTESTS');
```

Comment:
such condition split doesn't make sense , I would vote for 

`get_var('NVMFTESTS') && consolestep_is_applicable(); `  

doesn't matter if it will be 
`load_nvmftests                      if get_var('NVMFTESTS') && consolestep_is_applicable();`

or 
```
sub load_nvmftests {
     loadtest "kernel/nvmftests" if get_var('NVMFTESTS') && consolestep_is_applicable();
 }
```